example/DM-mentions/main.bal

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerinax/twitter;
+
+configurable string accessToken = ?;
+
+public function main() returns error? {
+    twitter:Client twitter = check new ({
+        auth: {
+            token: accessToken
+        }
+    });
+
+    twitter:Get2TweetsSearchRecentResponse supportTweets = check twitter->/tweets/search/recent(
+        query = "#ballerinaSupport",
+        max_results = 10
+    );
+
+    twitter:TweetId[] tweetIds = [];
+    twitter:UserId[] userIds = [];
+    twitter:Tweet[] tweets = supportTweets.data ?: [];
+
+    foreach twitter:Tweet tweet in tweets {
+        if tweet.id !is () {
+            tweetIds.push(check tweet.id.ensureType(string));
+        }
+    }
+
+    foreach twitter:TweetId tweetId in tweetIds {
```

Comment:
```suggestion
    twitter:UserId[] userIds = [];
    foreach twitter:TweetId tweetId in tweetIds {
```

Reply:
Solved in : [Remove unnecessary code in main.bal in examples](https://github.com/ballerina-platform/module-ballerinax-twitter/pull/111/commits/1d345526322b3ddd2af23349bba073dd80738b1d)